utilities/test_suite/rpp_test_suite_misc.h

```diff
@@ -261,9 +271,29 @@ std::map<Rpp32s, Rpp32u> paramStrideMap3D =
     {7, 3408}
 };
 
+// strides used for jumping to corresponding axisMask mean and stddev in 4D
+std::map<Rpp32s, Rpp32u> paramStrideMap4D =
+{
+    {1, 0}, // 4x10x25x40 - 4D
+    {2, 1000},
+    {3, 2600},
```

Comment:
This hardcoded map needs some comments as to why its 4, 10, 25, 40.
Is it for QA pass/fail tests based on current sample inputs being used? Please mention that